protocol/x/perpetuals/types/liquidity_tier_test.go

```diff
@@ -213,115 +199,39 @@ func TestLiquidityTierGetMaxAbsFundingClampPpm(t *testing.T) {
-			// initial margin * margin adjustment * quote quantums
+		"initial margin 20%": {
+			initialMarginPpm: uint32(200_000), // 20%
+			bigQuoteQuantums: big.NewInt(500_000),
+			// initial margin * quote quantums
 			// = 20% * 100% * 500_000
 			// = 100_000
 			expectedInitialMarginQuoteQuantums: big.NewInt(100_000),
 		},
-		"initial margin 50%, quote quantums == base position notional": {
-			initialMarginPpm:     uint32(500_000), // 50%
-			bigQuoteQuantums:     big.NewInt(1_000_000),
-			basePositionNotional: uint64(1_000_000),
-			// initial margin * margin adjustment * quote quantums
+		"initial margin 50%": {
+			initialMarginPpm: uint32(500_000), // 50%
+			bigQuoteQuantums: big.NewInt(1_000_000),
+			// initial margin * quote quantums
 			// = 50% * 100% * 1_000_000
 			// = 500_000
 			expectedInitialMarginQuoteQuantums: big.NewInt(500_000),
 		},
-		"initial margin 40%, quote quantums > base position notional": {
-			initialMarginPpm:     uint32(400_000), // 40%
-			bigQuoteQuantums:     big.NewInt(4_000_000),
-			basePositionNotional: uint64(1_000_000),
-			// initial margin * margin adjustment * quote quantums
-			// = 40% * sqrt(4) * 4_000_000
-			// = 3_200_000
-			expectedInitialMarginQuoteQuantums: big.NewInt(3_200_000),
-		},
-		"initial margin 60%, quote quantums > base position notional, adjusted margin capped at 100%": {
-			initialMarginPpm:     uint32(600_000), // 60%
-			bigQuoteQuantums:     big.NewInt(4_000_000),
-			basePositionNotional: uint64(1_000_000),
-			// initial margin * margin adjustment * quote quantums
-			// = 60% * sqrt(4) * 4_000_000
-			// = 100% * 4_000_000 (adjusted margin capped at 100%)
-			// = 4_000_000
-			expectedInitialMarginQuoteQuantums: big.NewInt(4_000_000),
-		},
-		"initial margin 0%, quote quantums > base position notional": {
-			initialMarginPpm:                   uint32(0), // 0%
-			bigQuoteQuantums:                   big.NewInt(1_000_000_000_000),
-			basePositionNotional:               uint64(1_000_000),
-			expectedInitialMarginQuoteQuantums: big.NewInt(0),
-		},
 		"initial margin 10%, quote quantums = 1, should round up to 1": {
-			initialMarginPpm:     uint32(100_000), // 10%
-			bigQuoteQuantums:     big.NewInt(1),
-			basePositionNotional: uint64(1_000_000),
+			initialMarginPpm: uint32(100_000), // 10%
+			bigQuoteQuantums: big.NewInt(1),
 			// initial margin * margin adjustment * quote quantums
 			// = 10% * 100% * 1
 			// = 0.1 -> round up to 1
 			expectedInitialMarginQuoteQuantums: big.NewInt(1),
 		},
 		"initial margin 56.7243%, quote quantums = 123_456, should round up to 70_030": {
-			initialMarginPpm:     uint32(567_243), // 56.7243%
-			bigQuoteQuantums:     big.NewInt(123_456),
-			basePositionNotional: uint64(1_000_000),
+			initialMarginPpm: uint32(567_243), // 56.7243%
+			bigQuoteQuantums: big.NewInt(123_456),
 			// initial margin * margin adjustment * quote quantums
 			// = 56.7243% * 100% * 123_456
 			// ~= 70029.5518 -> round up to 70030
```

Comment:
are you taking about this test specifically? from the comment below - the adjustment was 100% so the margin requirements are the same before and after 

Removed the comment since margin adjustment is no longer applicable, but pasted here as an FYI
```
			// initial margin * margin adjustment * quote quantums
			// = 56.7243% * 100% * 123_456
			// ~= 70029.5518 -> round up to 70030
```

There are a few tests that got updated I believe.